serve/main.go

```diff
@@ -1,326 +1,59 @@
 package main
 
 import (
-	"bytes"
-	"context"
-	"crypto/sha512"
-	"encoding/base64"
-	"encoding/json"
 	"fmt"
-	"io"
 	"log"
-	"math/rand"
 	"net/http"
-	"net/url"
 	"os"
-	"strings"
 	"time"
 
-	"cloud.google.com/go/storage"
-	"github.com/covidtrace/jwt"
-	"github.com/google/uuid"
+	"github.com/covidtrace/operator/handler"
+	"github.com/covidtrace/operator/storage"
+	"github.com/covidtrace/operator/util"
 	"github.com/julienschmidt/httprouter"
-	twilio "github.com/kevinburke/twilio-go"
 )
 
-func getEnvVar(key string) string {
-	value := os.Getenv(key)
-	if value == "" {
-		panic(fmt.Errorf("%s is required env var", key))
-	}
-
-	return value
-}
-
-var tokenNamespace string
-var tokenIssuer *jwt.Issuer
-var refreshTokenIssuer *jwt.Issuer
-var hashSalt string
-var codeLength int = 6
-
-var storageClient *storage.Client
-var storageBucket *storage.BucketHandle
-
-var twilioMessages *twilio.MessageService
-var twilioLookup *twilio.LookupPhoneNumbersService
-var twilioFromNumber string
+var tokenHandler handler.Handler
+var elevatedHandler handler.Handler
 
 func init() {
 	var err error
 
-	tokenNamespace = getEnvVar("JWT_NAMESPACE")
+	jsk := []byte(util.GetEnvVar("JWT_SIGNING_KEY"))
+	tns := util.GetEnvVar("JWT_NAMESPACE")
 
-	td, err := time.ParseDuration(getEnvVar("JWT_TOKEN_DURATION"))
+	td, err := time.ParseDuration(util.GetEnvVar("JWT_TOKEN_DURATION"))
 	if err != nil {
 		panic(err)
 	}
 
-	tokenIssuer = jwt.NewIssuer(
-		[]byte(getEnvVar("JWT_SIGNING_KEY")),
-		fmt.Sprintf("%s/operator", tokenNamespace),
-		fmt.Sprintf("%s/token", tokenNamespace),
-		td,
-	)
-
-	rd, err := time.ParseDuration(getEnvVar("JWT_REFRESH_DURATION"))
+	rd, err := time.ParseDuration(util.GetEnvVar("JWT_REFRESH_DURATION"))
```

Comment:
Does it make sense that both elevated and non-elevated JWTs have the same refresh duration?

Reply:
I think it's probably fine but I'll make them separate env vars.